Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:12 as build
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+
+RUN npm install
```

Comment:
You need to run both COPY commands before doing the first install, as the package.json depend on each other.

Reply:
If you look i copy and install in this directory and then i copy and install the calc directory then i build since it depends on the calc dir.
This is the same instructions from the doc and it works

Reply:
Yes, this would get around the no package-lock.json committed.
Why is package-lock.json being ignore in the .gitignore?